pypom/selenium_driver.py

```diff
@@ -0,0 +1,133 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from zope.interface import (
+    implementer,
+    Interface,
+)
+
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver import (
+    Remote,
+    Opera,
```

Comment:
Remove already covers Firefox and Chrome (if I remember well, I'll have a look at the splinter code another time, just in case)